index.js

```diff
@@ -4,6 +4,9 @@ var hasFlag = require('has-flag');
 var serializeError = require('serialize-error');
 var Runner = require('./lib/runner');
 var runner = new Runner();
+var cache = require('cacha')('.ava/cache');
+
+cache.clean();
```

Comment:
Does it do that every time `cache.clean()` is called? If yes, it might be worth only doing it once in awhile and checking with a file the last time it was done.